lib/TelemetryFeed.jsm

```diff
@@ -420,7 +415,6 @@ this.TelemetryFeed = class TelemetryFeed {
         source,
         tiles: impressionSets[source],
       });
-      this.sendEvent(payload);
```

Comment:
Technically, if I understand what's happening, it's probably harmless to leave this line here because `sendEvent` is now a "black hole" you're just removing this line as a little cleanup, right?

Reply:
Ah, so given that this is an event that's already been migrated, there is no need to ever have `sendEvent` here.

Reply:
`sendStructuredIngestionEvent` is the new pipeline now, right?

Reply:
Exactly, we used to send it to both sides, now there is no point to do so as it's already been migrated.